Misc/NEWS.d/next/Library/2020-05-08-21-30-54.bpo-40550.i7GWkb.rst

```diff
@@ -0,0 +1 @@
+Fix time-of-check/time-of-action issue in multiprocessing.Popen.send_signal.
```

Comment:
"multiprocessing.Popen.send_signal" is not valid.

Reply:
Ups, sorry.